ouroboros-consensus/src/unstable-consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/BitVector.hs

```diff
@@ -162,11 +163,17 @@ fillInWindow ::
   -> g
   -> C.MVector base SlotE s S
   -> ST s (C.Var base (PreImage pol ActiveSlotE))   -- ^ the count after filling
-fillInWindow pol (SomeDensityWindow k s) g mv = do
+fillInWindow pol (SomeDensityWindow k s) g mv
+        | not (C.getCount k <= C.getCount s) =
+            error $ "fillInWindow: assertion failure: k <= s: "
+                    ++ show k ++ " <= " ++ show s
+        | not (C.getCount sz <= C.getCount s) =
+            error $ "fillInWindow: assertion failure: sz <= s: "
+                    ++ show sz ++ " <= " ++ show s
+        | otherwise = do
     -- how many active polarized slots @actual@ currently has
     initialActives <- countActivesInMV pol mv
```

Comment:
Minor: I think the comment should talk about `@mv@` instead of `@actual@` (but fixing this does not have to happen in this PR, but is trivial enough that it also doesn't seem like a huge distraction).

Reply:
Adopted